### PR TITLE
Added Gary's Story

### LINKED ISSUE #4: Add Gary's Story
## Description
Add Garys's Story to the app.

The copy for the story is here: https://docs.google.com/document/d/1Bxti8B3casqFMXRlsFcIewaFkqU2hVVnjG_ZXZyC7S4/edit?usp=sharing

Folder for illustrations: https://drive.google.com/drive/folders/1Y4yh39QEfzXcu1vHws2sNNCl8ZM4L_RK?usp=sharing

## Plan
- [ ] Add Keys with Copy to BrandCopy.elm
- [ ] Add alt text if not there yet

## Acceptance Criteria 
- [x] Gary's story is listed on the stories
- [x] The copy matches the copy in the Google Doc - https://docs.google.com/document/d/1Bxti8B3casqFMXRlsFcIewaFkqU2hVVnjG_ZXZyC7S4/edit?usp=sharing
- [x] Illustrations match the ones in this folder: https://drive.google.com/drive/folders/1Y4yh39QEfzXcu1vHws2sNNCl8ZM4L_RK?usp=sharing
- [x] At the end of the story the bite size info shown is 5 and 6

### PR DESCRIPTION
**Pull request by HedgehogHolly**
Fixes #4 

## Description

- Added Gary's Story
- Missing two quotes from Gary's copy

@neontribe/contemplating-action

### CODE REVIEW

File: src/Copy/BrandCopy.elm [hunk 1]
```diff
@@ -464,7 +464,7 @@ brandCopy key =
             CopyText "After Hazel’s husband Colin was diagnosed with Progressive Supranuclear Palsy they both struggled to adjust and needed a little extra support."
 
         StoryTwoTeaserImageAlt ->
-            CopyText "\u{2728}"
+            CopyText "✨"
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: Would you mind also adding some alt text for all the images that don't have it? Looks like maybe is the teaser imaged.

Comment (HedgehogHolly) [reply src/Copy/BrandCopy.elm]: Fixed it :)

**Review by katjam — APPROVED** [review]
Great! :balloon:

**Review by rosejbon — APPROVED** [review]
Looks brilliant :)

**Pull request merged**